net/node/localnode.go

```diff
@@ -44,7 +45,8 @@ type LocalNode struct {
 
 	sync.RWMutex
 	syncOnce          *sync.Once
-	relayMessageCount uint64 // count how many messages node has relayed since start
+	relayMessageCount uint64    // count how many messages node has relayed since start
+	uptime            time.Time // UNIX timestamp of localNode init
```

Comment:
I think it's better to change the name "uptime" (duration) to "timeStarted" or "startTime" (time) here.

Reply:
Rename done.